sematic/plugins/building/docker_builder.py

```diff
@@ -463,20 +663,21 @@ def _push_image(
         )
         return _reload_image_uri(image=image)
 
-    for status_update in response:
+    for status_update in status_updates:
         if "error" in status_update:
```

Comment:
This seems slightly risky. What if the user has named their image `error_ingestor` or something of the like? The build would always fail, right? Is there a more robust regex we can use, or some other signal (like the contents of `"errorDetail"`)?

Reply:
Good catch! Updated the check to explicitly check the keys returned by the Docker server in the response.